.github/ISSUE_TEMPLATE/PLIP.md

```diff
@@ -1,10 +1,10 @@
 ---
 name: "\U0001F680 PLIP"
-about: Plone Improvement Proposal
+about: 'A Plone Improvement Proposal (PLIP) is a larger change to Plone, usually affecting multiple packages, and which goes through a formal process. See [Plone Improvement Proposals (PLIPs)](https://6.docs.plone.org/contributing/core/plips.html) for details.'
```

Comment:
It's not allowed to be this long.
```suggestion
about: 'A Plone Improvement Proposal (PLIP) is a larger change to Plone, usually affecting multiple packages, and which goes through a formal process.'
```

Reply:
@davisagli why did you remove the link to details?

Reply:
I explained why in the CMFPlone PR. 200 character limit